opentelemetry-tracing/opentelemetry-dotnet/README.md

```diff
@@ -0,0 +1,13 @@
+# OpenTelemetry .NET instrumentation examples
+
+This folder contains sample applications that were instrumented using
```

Comment:
I'd remove "that were", and just leave "This folder contains sample applications instrumented using..."